internal/component/configmap/configmap_test.go

```diff
@@ -377,8 +370,20 @@ func matchClientTLSRelatedConfiguration(g *WithT, etcd *druidv1alpha1.Etcd, actu
 	}
 }
 
+func getAdvertiseURLsInterface(etcd *druidv1alpha1.Etcd, commType, scheme string) map[string]interface{} {
+	advertiseUrlsMap := getAdvertiseURLs(etcd, commType, scheme, druidv1alpha1.GetPeerServiceName(etcd.ObjectMeta))
```

Comment:
`getAdvertiseURLsInterface` should not rely on `getAdvertiseURLs` directly, as this does not help in testing `getAdvertiseURLs` independently.